### DISCUSSION TITLE
Options for creating back-office editing modules link for end-users

### DISCUSSION THREAD

**Discussion by SemaphoreOxalis** [joomla/joomla-cms#47088] [2. Cold Features]
### Is your feature request related to a problem?
The whole things is about creating better end-user experience for editing specific modules.
For example the webmaster use some gallery, cumstom_content, homemade modules used for the displaying some data but the end-user need to edit them (not like menu or breadcrumbs).
The webmaster used the ACL for restricting the access to the critical modules (menu, breadcrumbs etc) but they still appear the "modules listing view". With small projects it's not that bothering but with medium/big ones modules listing view is complete mess and disturbing for the end user even with using the notes fields.

### Describe the solution you'd like
More options when, creating admin module link
- a link which make easier to link to a specific module in "editing mode" than creating a URL link and manually writing index.php?option=com_modules&task=module.edit&id=xx (I'm not sure it's a good thing to do)
- menu link for pre-filtering the modules view by text, type or position. For example we could use the notes for filtering on (the text search already used the notes field)

### Additional context
Yeah I know we have front-end editing but I'm looking for back-office solution here.
Let's discuss !

**Comment by brianteeman** [comment]
So you are looking for a way of _hiding_ modules from the list of modules if a user does not have permission to edit them?

**Comment by SemaphoreOxalis** [comment]
I already thought about hiding (and at least it could be that) but it didn't solve the case where we want to create 1 admin link to "all footer modules only" and an other to all "news flash module's type  and an other to a specific module with custom content.
I'm a bit ambitious/greedy but we aim to build very tailored back-office for the redactors ( and demonstrating that WP bo is not easier than J! ^^)

**Comment by brianteeman** [comment]
link filtered by position
administrator/index.php?option=com_modules&view=modules&client_id=0&filter[position]=POSITIONNAME

link filtered by type
administrator/index.php?option=com_modules&view=modules&client_id=0&filter[module]=MODULENAME

link filtered by search term
administrator/index.php?option=com_modules&view=modules&client_id=0&filter[search]=SEARCHTERM

**Comment by SemaphoreOxalis** [comment]
Thanks, and sorry for not being clear enough. Personally, I already know these "tricks". I was thinking for newcomers (or old ones not knowing how to play with URL variables). 
My proposition was to add option during the menu link creation (quite as for the articles, even if the articles don't the searchterm option). But maybe it's too much "a niche"  request.
At least the Issue will be find by ppl looking for these tricks.

**Comment by brianteeman** [comment]
It's definitely niche but maybe it will scratch someones itch

**Comment by SemaphoreOxalis** [comment]
After all this time, we find Quick icons from Joomill. It's perfect for designing end user dashboard.
